### ISSUE TITLE
Footer - 

### ISSUE THREAD

**Issue by HerinHentry** [govau/uikit#411]
Regarding Footer https://designsystem.gov.au/components/footer/

When there is more than one navigation area on the page, each should have an aria-label to distinguish them. Apparently, when there is a footer with a `<nav>`, this might be the second navigation area unless it's a login page or spotlight page. Never the less adding an aria-label is a good user experience.

Further reference:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/navigation.html

**Comment by HerinHentry** [comment]
This is the snapshot of how the current page looks in element list with multiple `<nav>`.

![image](https://user-images.githubusercontent.com/15111528/40815871-e0dbc43e-658c-11e8-90eb-db7f7753c1ea.png)

**Comment by HerinHentry** [comment]
Applies to Inpage links as well. When the H2 is used with aria-labelledby to give a unique name to this navigation area, this benefits SR users tabbing into the links.

`<nav class="au-inpage-nav-links">
  <h2 class="au-inpage-nav-links__heading">Contents</h2>
  <ul class="au-link-list">
    <li><a href="#section1">Section 1</a></li>
    <li><a href="#section2">Section 2</a></li>
  </ul>
</nav>`

**Comment by alex-page** [comment]
@HerinHentry this is again awesome feedback. 

As I mentioned before we are trying to keep the examples simple however this seems like quite a simple and logical change!

**Comment by alex-page** [comment]
We have published new versions of the components with `aria-label`. We are going to update the documentation on the design system website in https://github.com/govau/designsystem/issues/211 then this will be ready to be closed.
